scalableminds/webknossos
# Run fossildb in background

This adds:
- utilities to download and run fossildb, and
- a replacement script for sbt, that starts fossildb in the background.
The docker-compose services now use this script.

### Steps to test:
- run wk via `./sbt` or `docker-compose up dev`
- verify that fossildb is running (e.g. by looking into `fossildb/logs`)

### Issues:
- related to scalableminds/kube#65

------
